example/package.json

```diff
@@ -11,7 +11,7 @@
   },
   "dependencies": {
     "react": "19.1.0",
```

Comment:
BTW We don't support React Native 0.78/0.79 with React 19.1.0

The version of React is dictated by React Native. If you bump it here is effectively ignored. More on this here: https://github.com/facebook/react-native/blob/main/packages/react-native/Libraries/Renderer/README.md